package.json

```diff
@@ -3,8 +3,8 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "start": "npm run dev",
-    "dev": "vite dev --port 3000",
+    "start": "node ./build",
```

Comment:
I really enjoy just running `npm start` instead of `npm run dev`, could you leave this script and add a new one (e.g. prod) for this?